articles/flutter-vs-kotlin/index.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/flutter-vs-kotlin/
+title: Flutter vs. Kotlin
+description: This article goes over a comparison of Flutter and Kotlin - talking the advantages and disadvantages of both.
+author: Bavneet Singh
+date: 2020-09-01T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/flutter-vs-kotlin/hero.jpg
+    alt: Kotlin vs Flutter image example
+---
+<!--more-->
+Mobile application development has become an essential part of the software business in recent years. Developers used to prefer ReactNative (or Flutter to a lesser extent) to develop cross-platform applications due to their low-cost features. Today, another cross-platform language named Kotlin has also managed to enter into the world of cross-platform development. Since then Flutter has also managed to improve a lot in the past years, and it now provides better features than ReactNative.  
+
+In this article, we will be comparing Flutter & Kotlin as cross-platform development languages. The primary goal of this article is to help developers distinguish the best cross-development platform for them, depending on their needs, and skills.
+
+### Performance
+Kotlin and Flutter, are superb choices when it comes to mobile app development, both with their own distinctions. Kotlin can compile the code and customize it to the best needs of its user as well as the native apps.
+
+Flutter has a reloading functionality, that gives the developer a choice of selection if all the different elements are performing well.
+
+Developers describe Flutter as one of the most popular cross-platform mobile framework. Kotlin, however, is described more as a statistical-type programming language.
+
+### Language
+One of the main advantages of Kotlin is that its programming language is quite similar to Java. Therefore developers who have worked with Java in the past will find it much easier to work with Kotlin. 
+On the other hand, Flutter uses Dart from Google. Due to the fact that Dart is a client-optimized programming language, it may offer an easier option for developers who have strong knowledge of C languages. It is easier to understand and operate due to fast compiling like C languages.
+
+### Price and Testing
+Kotlin and Flutter are both free, but the real difference occurs in testing support. While Kotlin tends to support backend development, Flutter works on extensive testing of features.
```

Comment:
More detail needed on how Kotlin supports backend development and what testing features Flutter enables.